### PR TITLE
Filters UI

### PR DESCRIPTION
**Pull request by JonasPfeifer05**

### CODE REVIEW

File: components/FilterSettings.vue [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+        </div>
+      </div>
+      <div id="filter-footer">
+        <button id="reset-button" @click="resetFilters">reset</button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style scoped lang="scss">
```
Comment (ToBinio) [inline components/FilterSettings.vue:166]: please use the Loggaroo [colors](https://project-4ahif-jp-fg-bh-tf.github.io/wiki/)

note:
there is some global CSS with variables for the colors

**Comment by JonasPfeifer05** [comment]
Ok but why didnt the error occure before?
And what are we supposed to do about it?

File: components/TableView.vue [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
 
 const entryStore = useLogEntryStore();
+const orderStore = useOrderStore();
 
 callOnce(() => {
-  entryStore.loadEntries(0, 1);
+  entryStore.loadNextEntries();
+});
+
+// TODO use backend
```
[the remark this answers is not in the document]
Comment (JonasPfeifer05) [reply components/TableView.vue:12]: Added a new task to taiga

File: components/FilterSettings.vue (hunk 1 above)
Comment (JonasPfeifer05) [reply components/FilterSettings.vue:166]: Forgot thanks

File: stores/orderStore.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { defineStore } from "pinia";
+
+export interface State {
```
[the remark this answers is not in the document]
Comment (JonasPfeifer05) [reply stores/orderStore.ts:3]: LogEntryStore accesses the order variable so it needs to be exported right?

**Comment by ToBinio** [comment]
I believe because we did not use the enum out side so the compiler just ignored it

Rename the file to `LogEntry.ts`

File: stores/orderStore.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (ToBinio) [reply stores/orderStore.ts:3]: the `state` of the LogEntryStore isn't exported either and it works fine so I don't think it's necessary

**Comment by JonasPfeifer05** [comment]
Ok

**Comment by JonasPfeifer05** [comment]
done

File: stores/orderStore.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (JonasPfeifer05) [reply stores/orderStore.ts:3]: ok

**Review by ToBinio — APPROVED** [review]
LGTM

**Pull request merged**